src/DynamoCore/BuiltInAndOperators/BuiltIn.Migrations.xml

```diff
@@ -72,6 +72,16 @@
       </attribute>
     </additionalAttributes>
   </priorNameHint>
+  <priorNameHint>
+    <oldName>RemoveNulls</oldName>
+    <newName>DSCore.List.Clean</newName>
+    <additionalAttributes>
+      <attribute>
+        <name>preserveIndices</name>
```

Comment:
@aparajit-pratap what does this do? (default values?)

Reply:
It migrates the node and sets the default argument value to `false` as the default value parameter is actually set to `true`.